framework/set/resources/airgap/rancher/setupAirgapRancher.go

```diff
@@ -19,12 +19,17 @@ const (
 // CreateAirgapRancher is a function that will set the airgap Rancher configurations in the main.tf file.
 func CreateAirgapRancher(file *os.File, newFile *hclwrite.File, rootBody *hclwrite.Body, terraformConfig *config.TerraformConfig,
 	rke2BastionPublicDNS, registryPublicDNS string) (*os.File, error) {
-	userDir, err := os.UserHomeDir()
-	if err != nil {
-		return nil, err
+	var err error
+	userDir := os.Getenv("GOROOT")
+	if userDir == "" {
+		userDir, err = os.UserHomeDir()
+		if err != nil {
+			return nil, err
+		}
+		userDir = filepath.Join(userDir, "go/")
```

Comment:
Just a small thing. For code readability, can you add a new line after the if condition? This would be for all of the other files as well.